modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/DDMTemplateLocalServiceTest.java

```diff
@@ -504,16 +506,24 @@ public void testSearchCountByResourceClassNameId() throws Exception {
 
 	@Test
 	public void testSmallImageWithInvalidURL() throws Exception {
-		DDMTemplate template = addTemplate(
-			_classNameId, 0, _resourceClassNameId, StringUtil.randomString(),
-			StringUtil.randomString(), StringUtil.randomString(),
-			DDMTemplateConstants.TEMPLATE_TYPE_FORM,
-			DDMTemplateConstants.TEMPLATE_MODE_CREATE,
-			TemplateConstants.LANG_TYPE_VM,
-			getTestTemplateScript(TemplateConstants.LANG_TYPE_VM),
-			WorkflowConstants.STATUS_APPROVED, true, "foo");
+		try {
+			addTemplate(
+				_classNameId, 0, _resourceClassNameId,
+				StringUtil.randomString(), StringUtil.randomString(),
+				StringUtil.randomString(),
+				DDMTemplateConstants.TEMPLATE_TYPE_FORM,
+				DDMTemplateConstants.TEMPLATE_MODE_CREATE,
+				TemplateConstants.LANG_TYPE_VM,
+				getTestTemplateScript(TemplateConstants.LANG_TYPE_VM),
+				WorkflowConstants.STATUS_APPROVED, true, "foo");
 
-		Assert.assertFalse(template.isSmallImage());
+			Assert.fail();
+		}
+		catch (PortalException portalException) {
+			Assert.assertTrue(
+				portalException instanceof
+					TemplateSmallImageInvalidURLException);
+		}
 	}
```

Comment:
@MrJohn1911 how about simplifying by using the `expected` attribute of the `@Test` annotation?

```
@Test(expected = TemplateSmallImageURLException.class)
public void testSmallImageWithInvalidURL() throws Exception {
	addTemplate(
		_classNameId, 0, _resourceClassNameId, StringUtil.randomString(),
		StringUtil.randomString(), StringUtil.randomString(),
		DDMTemplateConstants.TEMPLATE_TYPE_FORM,
		DDMTemplateConstants.TEMPLATE_MODE_CREATE,
		TemplateConstants.LANG_TYPE_VM,
		getTestTemplateScript(TemplateConstants.LANG_TYPE_VM),
		WorkflowConstants.STATUS_APPROVED, true, "foo");
}
```

Reply:
Ok, I will do this!